socorro/cron/jobs/bugzilla.py

```diff
@@ -186,3 +179,52 @@ def _iterator(self, from_date):
                 int(report['id']),
                 find_signatures(report.get('cf_crash_signature', ''))
             )
+
+
+class BugzillaCronAppDryRunner(App):  # pragma: no cover
```

Comment:
I tested running bugzilla-associations via crontabber and via this class and I don't think I'll ever use this class. It's much easier to run crontabber and specify the job in the dev environment which picks up configuration from the environment than it is to run this class with the additional arguments.

```
$ socorro/cron/crontabber_app.py --job=bugzilla-associations
```

vs.

```
$  python socorro/cron/jobs/bugzilla.py --database.user=postgres --database.password=aPassword --database.dbname=breakpad
```